lib/howl/interactions/line_selection.moon

```diff
@@ -49,13 +49,24 @@ line_match_highlighter = (editor) ->
             continue
           line = buffer.lines[nr]
           start_pos = line.start_pos
-          segments = Matcher.explain text, line.text
+          segments = explain text, line.text
           if segments
             for segment in *segments
               ranges[#ranges + 1] = { start_pos + segment[1] - 1, segment[2] }
 
         highlight.apply 'search_secondary', buffer, ranges
 
+create_matcher = (find) ->
+  class CustomMatcher
```

Comment:
Not a big fan on the in-line class definition here. There seems to be need for it as well? The class could be moved up to top-level, and take both `line_items` and `find` as constructor arguments.